class.jetpack-gutenberg.php

```diff
@@ -89,6 +102,39 @@ class Jetpack_Gutenberg {
 	 */
 	private static $availability = array();
 
+	/**
+	 * Check to see if a block is able to run with current wp or gutenburg plugin versions
+	 *
+	 * @param string $core_wp_version The minimum wp core version needed to run the block.
+	 * @param string $plugin_version The minimum gutenberg plugin version needed to run the block.
+	 *
+	 * @since 8.4.0
+	 *
+	 * @return boolean True if the block is gated due to current wp or plugin versions.
+	 */
+	public static function is_block_version_gated( $core_wp_version, $plugin_version ) {
+		global $wp_version;
+
+		if ( defined( 'IS_WPCOM' ) && IS_WPCOM ) {
```

Comment:
Does this check `true` on Atomic sites? (Those can sometimes run older versions of Gutenberg.)

Reply:
good question, will try and find out

Reply:
Apparently this will only return true for simple sites.